app/src/main/java/org/simple/clinic/editpatient/EditPatientViewRenderer.kt

```diff
@@ -82,7 +82,7 @@ class EditPatientViewRenderer(private val ui: EditPatientUi) : ViewRenderer<Edit
     val newlyAddedBpPassports = model.ongoingEntry.bpPassports?.map { it.displayValue() }.orEmpty()
     val identifiers = model.bpPassports?.map { it.identifier.displayValue() }.orEmpty()
 
-    ui.displayBpPassports(identifiers + newlyAddedBpPassports)
+    ui.displayBpPassports(identifiers, newlyAddedBpPassports)
```

Comment:
This approach works fine, but we end up passing two lists and then checking the highlight status at the view level. 

Instead, maybe we can map the identifiers to a specific type and then render those? Something like this.

```kt
val newlyAddedBpPassports = model.ongoingEntry.bpPassports
    ?.map { BpPassportListItem(identifier = it.displayValue(), isHighlighted = true) }
    .orEmpty()

val identifiers = model.bpPassports
    ?.map {
      BpPassportListItem(identifier = it.identifier.displayValue(), isHighlighted = false)
    }
    .orEmpty()
```

So, now we don't have to use nested loops to identify the highlight status at view level and can simply use the boolean value. This way we can actually have the tests to verify the highlight status as well. WDYT?